tests/test_integrations.py

```diff
@@ -148,3 +148,53 @@ def test_pycocotools():
     validator.is_coco = True
     download(f"{url}person_keypoints_val2017.json", dir=DATASETS_DIR / "coco8-pose/annotations")
     _ = validator.eval_json(validator.stats)
+
+
+@pytest.mark.skipif(not check_requirements("faster-coco-eval", install=False), reason="faster-coco-eval not installed")
```

Comment:
I don't know, maybe we can combine this test with the previous one. Just set @parameterized.expand